beacon-chain/db/kv/validators.go

```diff
@@ -0,0 +1,49 @@
+package kv
+
+import (
+	"context"
+
+	pb "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+)
+
+// ValidatorLatestVote retrieval by validator index.
+// TODO(#3164): Implement.
+func (k *Store) ValidatorLatestVote(ctx context.Context, validatorIdx uint64) (*pb.ValidatorLatestVote, error) {
```

Comment:
what do you think of just using `idx` or `index` instead of `validatorIdx`?

Reply:
I want to be as explicit as possible and leave no room for ambiguity, same with blockRoot instead of root as there can be many different types of roots and I want to make it clear what we refer to

Reply:
sgtm